### DISCUSSION TITLE
App check still use re-captcha on Android

### DISCUSSION THREAD

**Discussion by minhnguyenit14** [invertase/react-native-firebase#6970] [Q&A]
I set up everything as the docs recommend with correct project linking and correct SHA-256 mapping. I got the token from `getToken` API but it still opens the browser for re-captcha when I use the phone auth service. I've searched for hours and also checked on an emulator, android device debug build, assembleRelease build, and even from Play Store internal testing.
I don't know what's going on...

```
react-native: 0.63.5
@react-native-firebase/app: "17.3.2"
@react-native-firebase/auth: "17.3.2"
@react-native-firebase/app-check: "17.3.2"
```

- google play console
![image](https://user-images.githubusercontent.com/38130019/223690552-91c1d9a4-79b8-41d1-9b54-baee435a4462.png)
- google cloud
![image](https://user-images.githubusercontent.com/38130019/223690593-1110d72a-8aa7-4d5a-abd5-716a614d2798.png)
- firebase
![image](https://user-images.githubusercontent.com/38130019/223690713-546ab841-6fb8-42a5-b49a-98ae6739e727.png)
- in-app
![image](https://user-images.githubusercontent.com/38130019/223691555-9830c2b3-1b7c-407d-9730-65881a2d51a6.png)

**Comment by mikehardy** [comment]
I don't believe this version of react-native still receives support:

> react-native: 0.63.5

In general we follow this: https://github.com/reactwg/react-native-releases#releases-support-policy

I think you need to test on a real device in release mode and you need to make sure the SHA you are using is the one from Play Store (after they reprocess it)

If you've done all that I'm not sure what could be going wrong, sorry
